From 8c04c52ca6517792035eb7c5b7704f6cf1a1b5c8 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sat, 16 Dec 2023 22:50:34 -0800
Subject: [PATCH 2/2] VVI-OpenCL-CLHPP : Patch to add BUILD bazel

Summary:
Add BUILD to make it bzlmod compatible

Test Plan:
integration with vvplatform
---
 BUILD | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 BUILD

diff --git a/BUILD b/BUILD
new file mode 100644
index 0000000..cd4579d
--- /dev/null
+++ b/BUILD
@@ -0,0 +1,14 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+licenses(["notice"])
+
+exports_files(["LICENSE.txt"])
+
+cc_library(
+    name = "opencl_clhpp",
+    hdrs = glob([
+        "include/CL/*.hpp",
+    ]),
+    includes = ["include"],
+    visibility = ["//visibility:public"],
+)
\ No newline at end of file
-- 
2.34.1

